lib/experiments-page.php

```diff
@@ -51,6 +51,17 @@ function gutenberg_initialize_experiments_settings() {
 			'id'    => 'gutenberg-navigation',
 		)
 	);
+	add_settings_field(
+		'gutenberg-list-v2',
+		__( 'List block v2', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Enable List block v2 (with inner blocks)', 'gutenberg' ),
```

Comment:
Since this experiment will break list blocks / isn't ready for public consumption, should the label indicate that it's not ready for use? (I doubt many people check the experiments section, but just thinking of how we set expectations surrounding experiments)

We used to have a pretty big warning in the old gallery label (here's the PR that removed it: https://github.com/WordPress/gutenberg/pull/34979)